### PR TITLE
IDEM-27

### PR DESCRIPTION
**Pull request by jrocca82**
Screenshot 2022-05-30 132423

### CODE REVIEW

File: app/src/context/DocumentStore.tsx [hunk 1]
```diff
@@ -52,6 +54,10 @@ export const DocumentProvider: React.FC<{
       Crypto.CryptoDigestAlgorithm.SHA256,
       file.base64
     );
+    
+    let buffer = Buffer.from(file.base64, "base64");
```
Comment (bitcoinbrisbane) [inline app/src/context/DocumentStore.tsx:58]: Could this be a const?

```suggestion
    const buffer = Buffer.from(file.base64, "base64");
```

File: app/src/context/DocumentStore.tsx [hunk 2]
```diff
@@ -52,6 +54,10 @@ export const DocumentProvider: React.FC<{
...
+    const keccakHash = keccak256(buffer);
+    console.log(keccakHash);
```
Comment (bitcoinbrisbane) [inline app/src/context/DocumentStore.tsx:60]: lets remove?

**Review by bitcoinbrisbane — COMMENTED** [review]
2 small things

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by amagon270 — DISMISSED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]
LGTM

**Review by amagon270 — APPROVED** [review]

**Pull request merged**